src/lib.rs

```diff
@@ -87,35 +87,44 @@ pub trait Storage {
 }
 
 
-/// This is the encryption object and all file handling should be done via this as the low level 
+/// This is the encryption object and all file handling should be done via this as the low level
 /// mechanism to read and write *content* this library has no knowledge of file metadata. This is
-/// a library to ensure content is secured 
+/// a library to ensure content is secured
+
 pub struct SelfEncryptor<'a> {
   storage: &'a mut (Storage + 'a),
   my_datamap: datamap::DataMap,
   chunks: Vec<Chunks>,
   sequencer: Vec<u8>,
-  tempdir : TempDir, 
-  file_size: u64,
+  tempdir : TempDir,
+  pub file_size: u64,
```

Comment:
Is this dangerous Leo as somebody may change it from outside the functions we provide ? Perhaps there is another way to return just a copy of this number ?